### PR TITLE
zh:remove outdated files in /setup/upgrade/

### PR DESCRIPTION
**Pull request by windsonsea**
In the `content/zh/docs/setup/upgrade/` folder, removed 3 files:
```
content/zh/docs/setup/upgrade/cni-helm-upgrade/index.md
content/zh/docs/setup/upgrade/gateways/index.md
content/zh/docs/setup/upgrade/istioctl-upgrade/index.md
```

- [x] Docs

### CODE REVIEW

File: content/zh/docs/setup/upgrade/helm/revlabels_test.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```
Comment (zirain) [inline content/zh/docs/setup/upgrade/helm/revlabels_test.sh:1]: should not to sync this test script file, CI will not run test in `content/en`

Comment (windsonsea) [reply content/zh/docs/setup/upgrade/helm/revlabels_test.sh]: Thanks. This PR is a little big. I'm struggling in the errors -_! Thanks.

Comment (windsonsea) [reply content/zh/docs/setup/upgrade/helm/revlabels_test.sh]: Let me try to divide this PR to a smaller one before your review.

Comment (windsonsea) [reply content/zh/docs/setup/upgrade/helm/revlabels_test.sh]: Done! Removal in a PR and added localization in the other PR now. 😄

**Review by zirain — APPROVED** [review]

**Pull request merged**